src/binary/var_uint.rs

```diff
@@ -1,6 +1,7 @@
 use crate::data_source::IonDataSource;
```

Comment:
In this file:
* Adding write support for [the low-level encoding primitive `VarUInt`](https://amzn.github.io/ion-docs/docs/binary.html#varuint-and-varint-fields).